jest.config.js

```diff
@@ -2,4 +2,8 @@
 
 module.exports = {
   setupFiles: ['./test/setup.js'],
+  clearMocks: true,
```

Comment:
https://facebook.github.io/jest/docs/en/mock-function-api.html#mockfnmockclear , it think clearMock is the one required. I got few error of mocks calls not getting reset properly between calls.

Reply:
Interesting! Let's chat on monday